patchwork-events-world/src/main/java/com/patchworkmc/impl/event/world/IGetWorldFromChunkGenerator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2019, 2019
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package com.patchworkmc.impl.event.world;
+
+import net.minecraft.world.IWorld;
+
+public interface IGetWorldFromChunkGenerator {
+	IWorld getWorld();
```

Comment:
Perhaps instead of a seperate interface, this should be an interface mixin + @Accessor? probably definately. (i'm still learning new things about mixins)